### ISSUE TITLE
GUI: takes 120% of screen

### ISSUE THREAD

**Issue by mendomusic** [Vampouille/superboucle#45]
running the program from command line on Mint with conda python, the window is about 120% larger than my screen.

--> I cannot see the main menu.

**Comment by Vampouille** [comment]
Window size is fixed by size song : each clip take a fixed size. What is your screen definition ? I agree with you, clip should be resize in order to fit screen.
As a workaround, you can hold 'alt' key and then drag & drop main window (with alt key you should be able to move main window even if you don't see menu bar).Then you can use menu to define new song with less clips, for example 7x7 or 8x7.

**Comment by mendomusic** [comment]
>What is your screen definition ? 

1366x768

>I agree with you, clip should be resize in order to fit screen.

this would be a great thing.

>Then you can use menu to define new song with less clips, for example 7x7 or 8x7.

Possible but still leaves the window too large.

**Comment by Vampouille** [comment]
> Possible but still leaves the window too large.

even with little song like 4x4 ?

**Comment by milkmiruku** [comment]
I'm in a similar situation. I'm wondering how to reduce the song size though? I can't seem to find any menu option for that.

**Comment by Vampouille** [comment]
> I'm in a similar situation. I'm wondering how to reduce the song size though? I can't seem to find any menu option for that.

You can change song size when creating a new one. You can create a new song by using the menu.

**Comment by milkmiruku** [comment]
Aah, thanks, though it is indeed still too tall a window for my screen due to the sidebar;

![rose_2018-06-28-12 05 59](https://user-images.githubusercontent.com/108225/42030761-c7f0b9b2-7acb-11e8-9798-6b04ccbbe151.png)

**Comment by Vampouille** [comment]
> still too tall a window for my screen due to the sidebar;

yes, this may explain why some labels are truncated. what is the size of your screen ? There still some work to make superboucle able to fit on some resolution.

**Comment by milkmiruku** [comment]
Here is how the app looks without the theme, slightly different spacing but still a fair bit of it;  ![rose_2018-06-28-12 42 17](https://user-images.githubusercontent.com/108225/42032204-b955945e-7ad0-11e8-98c4-26a982c2d3e0.png)

Resolution is 1366x768.

**Comment by Vampouille** [comment]
It's better ! I think it need some work to handle QT Theme. As a workaround, you can move the windows upper by holding 'alt' key and drag and drop on the middle of the window. This way you should be able to access the lower part of the window.

**Comment by Vampouille** [comment]
I create a PR #52 to reduce height of left side bar. Now it fits on 1366x768 screen. Still need some work to reduce the main window height and adapt window size based on song size.
@milkmiruku you can test #52 that should fit on your laptop.

**Comment by milkmiruku** [comment]
Looks like this now; ![rose_2018-07-05-20 59 42](https://user-images.githubusercontent.com/108225/42345177-6fa5a3dc-8096-11e8-9687-0d0f706aae0a.png)

N.b. there's also the minimum size for the window that's not linked to the number of rows;

![rose_2018-07-05-21 01 03](https://user-images.githubusercontent.com/108225/42345272-b68e11f8-8096-11e8-9bd3-45392fe579a9.png)

**Comment by Vampouille** [comment]
> N.b. there's also the minimum size for the window that's not linked to the number of rows;

yes, I need to work on this.

**Comment by Vampouille** [comment]
This PR #58 resize the cell to reduce height of the window, it also remove the status bar. It should also fix the resize of the window.
It does not include #52, I need to take changes on the left side bar.

**Comment by trebmuh** [comment]
Hi, I'm on a similar situation. the height of SB window is too much for my screen. I can use the `Alt` workaround but it'd be great if SB would be able to fit with any screen.
